From 4d5908c8407e489ece3e415a7d4955956d040636 Mon Sep 17 00:00:00 2001
From: Shirley Zhao <shirley.zhao@intel.com>
Date: Mon, 5 Nov 2018 08:44:56 +0000
Subject: [PATCH] [TPMSimulator]fix compile issue for clear linux

---
 src/makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/makefile b/src/makefile
index cc3e410..607fcfc 100644
--- a/src/makefile
+++ b/src/makefile
@@ -81,3 +81,7 @@ clean:
 %.o:		%.c
 		$(CC) $(CCFLAGS) $< -o $@
 
+install:	tpm_server
+		install -d $(DESTDIR)/usr/bin
+		install -t $(DESTDIR)/usr/bin $<
-- 
2.24.0

